<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormResultComparer</class>
 <widget class="QWidget" name="FormResultComparer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditGroundTruth">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Basis-Datensatz (GT):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Vergleichs-Datensatz:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditCompare">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonLoadGroundTruth">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="qttestvs10.qrc">
         <normaloff>:/QTTestVs10/open.png</normaloff>:/QTTestVs10/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButtonLoadCompare">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="qttestvs10.qrc">
         <normaloff>:/QTTestVs10/open.png</normaloff>:/QTTestVs10/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="qttestvs10.qrc">
         <normaloff>:/QTTestVs10/tracker.png</normaloff>:/QTTestVs10/tracker.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>527</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxMassCenters">
         <property name="title">
          <string>Positions-Abweichung (Massenzentrum)</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Minimum:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Mittelwert:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Maximum:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Median:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditMassMin">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditMassMax">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditMassMid">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditMassMed">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSpinepoints">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Positions-Abweichung (Spine-Punkte)</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Minimum:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Maximum:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Mittelwert:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Median:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditSpineMin">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditSpineMax">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditSpineMid">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditSpineMed">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxWidths">
         <property name="title">
          <string>Abweichung der Larvenbreite</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLabel" name="label_19">
            <property name="toolTip">
             <string>Absoluter Breitenunterschied zwischen Vergleichs-Larven und Ground-Truth-Larven</string>
            </property>
            <property name="text">
             <string>Absolut:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Minimum:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditWidthMinAbs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Absoluter Breitenunterschied zwischen Vergleichs-Larven und Ground-Truth-Larven</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Maximum:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditWidthMaxAbs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Absoluter Breitenunterschied zwischen Vergleichs-Larven und Ground-Truth-Larven</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Mittelwert</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditWidthMidAbs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Absoluter Breitenunterschied zwischen Vergleichs-Larven und Ground-Truth-Larven</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Median:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEditWidthMedAbs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Absoluter Breitenunterschied zwischen Vergleichs-Larven und Ground-Truth-Larven</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_20">
            <property name="toolTip">
             <string>Vergleichs-Larve kleiner als Ground-Truth-Larve</string>
            </property>
            <property name="text">
             <string>Unterschnitt:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_21">
            <property name="toolTip">
             <string>Vergleichs-Larve größer als Ground-Truth-Larve</string>
            </property>
            <property name="text">
             <string>Überschnitt:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_22">
            <property name="toolTip">
             <string>Gegenrechnung zu großer und zu kleiner Modelle.</string>
            </property>
            <property name="text">
             <string>Generell:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEditWidthMinGen">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Gegenrechnung zu großer und zu kleiner Modelle.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEditWidthMaxGen">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Gegenrechnung zu großer und zu kleiner Modelle.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEditWidthMidGen">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Gegenrechnung zu großer und zu kleiner Modelle.</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="lineEditWidthMedGen">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Gegenrechnung zu großer und zu kleiner Modelle.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditWidthMinSmaller">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve kleiner als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEditWidthMidSmaller">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve kleiner als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEditWidthMaxSmaller">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve kleiner als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="lineEditWidthMedSmaller">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve kleiner als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QLineEdit" name="lineEditWidthMedBigger">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve größer als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="lineEditWidthMidBigger">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve größer als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEditWidthMaxBigger">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve größer als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="lineEditWidthMinBigger">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Vergleichs-Larve größer als Ground-Truth-Larve</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWorkingAnimation">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="qttestvs10.qrc">:/QTTestVs10/loader.gif</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qttestvs10.qrc"/>
 </resources>
 <connections/>
</ui>
